db-engine/src/main/java/com/splicemachine/db/catalog/SystemProcedures.java

```diff
@@ -1594,74 +1594,6 @@ public static void SYSCS_UPDATE_METADATA_STORED_STATEMENTS()
         }
     }
 
-    /**
-     * Create a system stored procedure.
-     * PLEASE NOTE:
-     * This method is currently not used, but will be used when Splice Machine has a SYS_DEBUG schema available
-     * with tools to debug and repair databases and data dictionaries.
-     *
-     * @param schemaName name of the system schema
-     * @param procName   name of the system stored procedure
-     * @throws SQLException
-     */
-    public static void SYSCS_CREATE_SYSTEM_PROCEDURE(String schemaName,String procName)
```

Comment:
I understand this is not used at the moment, but looking at the comment, it looks like we would need it once we have `SYS_DEBUG` schema available, is this still relevant or not?

Reply:
I don't think there are plans to bring sys_debug at the moment.